docs/getting_started.md

```diff
@@ -79,6 +79,10 @@ ILLIXR can also benchmark each component in isolation.
 
 3. Each component is a directory in `benchmark`. See those components for their documentation.
 
+## Virtual Machine
+
+ILLIXR can be run inside a Qemu-KVM image. Check out the instructions [here](../qemu/INSTRUCTIONS.md)
```

Comment:
Similar change here, this is going to be on our getting started docs webpage so this link needs to be the actual github URL to the INSTRUCTIONS.md

Reply:
Or move the qemu instructions to the docs folder and we can generate the webpage using mkdocs.

Reply:
You will also need to update the mkdocs.yml if you go this direction.